azerothcore/azerothcore-wotlk
# fix(Core/Formations): fixed possible finite loop related to GROUP_AI_FLAG_EVADE_TOGETHER

## Changes Proposed:
-  Fixes instability issue related to #9269
-  Minor revert to 

## Issues Addressed:

- Closes #9269

## SOURCE:


## Tests Performed:

- Tested mainly with this formation group loader group (NPC spawnId: 202212)
- 


## How to Test the Changes:


1. Complete Garfrost & Eck & Krick encounters
2. Go to tunnel (manually)
3. Once you reached tunnel, type .go creature 202212
4. Use Feign Death or Shadowmeld and observe it - npcs should evade without causing server to hang (crash)

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
